src/graphVisualization/Explorer/DefaultPanelContent/DefaultDetailsPane.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import React, { useState } from 'react'
+
+import {
+  GraphStyleModel,
+  NodeItem,
+  RelationshipItem
+} from 'graph-visualization'
+
+import { ClipboardCopier, PropertiesTable, upperFirst } from 'common'
+import { PaneBody, PaneHeader, PaneTitle } from '../styled'
+import { NodeLabel } from './NodeLabel'
+import { RelType } from './RelType'
+
+export const DETAILS_PANE_STEP_SIZE = 1000
+export type DetailsPaneComponentProps = {
+  vizItem: NodeItem | RelationshipItem
+  graphStyle: GraphStyleModel
+  nodeInspectorWidth: number
+}
+export function DetailsPaneComponent({
+  vizItem,
+  graphStyle,
+  nodeInspectorWidth
+}: DetailsPaneComponentProps): JSX.Element {
+  const [maxPropertiesCount, setMaxPropertiesCount] = useState(
+    DETAILS_PANE_STEP_SIZE
+  )
+
+  /* Add id to property list */
```

Comment:
Here is is again, still a bit unsure what you mean with this comment. Did you add it?